mod/05_email/set/right/bcc.rb

```diff
@@ -18,7 +22,7 @@ def process_email_addresses context_card, format_args, args
 format :email_text do
   view :email_addresses do |args|
     context = args[:context] || self
-    card.item_names.map do |item_name|
+    card.item_names(:context=>context.cardname).map do |item_name|
```

Comment:
One of the new tests failed without that. I think it was about a search card with contextual names in the query.
We need the next line for non-pointer cards: https://github.com/wagn/wagn/blob/master/mod/01_core/set/all/collection.rb#L40